resources/views/vendor/spark/auth/registration/subscription.blade.php

```diff
@@ -0,0 +1,57 @@
+@extends('spark::layouts.app')
+
+<!-- Scripts -->
+@section('scripts')
+	<script src="https://js.stripe.com/v2/"></script>
+
+	<script>
+		STRIPE_KEY = '{{ config('services.stripe.key') }}';
+	</script>
+
+	<script src="//cdnjs.cloudflare.com/ajax/libs/URI.js/1.15.2/URI.min.js"></script>
```

Comment:
That's the default that stripe uses. We can include it in the app if you
want.
On Nov 23, 2015 7:26 AM, "Ryan Raub" <notifications@github.com> wrote:

> In resources/views/vendor/spark/auth/registration/subscription.blade.php
> <https://github.com/gios-asu/gather-up-web/pull/4#discussion_r45607766>:
>
> > @@ -0,0 +1,57 @@
> > +@extends('spark::layouts.app')
> > +
> > +<!-- Scripts -->
> > +@section('scripts')
> > +	<script src="https://js.stripe.com/v2/"></script>
> > +
> > +	<script>
> > +		STRIPE_KEY = '{{ config('services.stripe.key') }}';
> > +	</script>
> > +
> > +	<script src="//cdnjs.cloudflare.com/ajax/libs/URI.js/1.15.2/URI.min.js"></script>
>
> do we really want to use a cdn for this asset?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gios-asu/gather-up-web/pull/4/files#r45607766>.
>